### PR TITLE
UX-1749/Login Page Fixes

### PR DESCRIPTION
**Pull request by Nalinmujoo**
Login Page Fixes
https://user-images.githubusercontent.com/31595960/163667869-67150457-6d6e-4231-9b92-8a5e23cee38a.mov

### CODE REVIEW

File: src/app/core/public/LoginPage.tsx [hunk 1]
```diff
@@ -173,7 +176,7 @@ const LoginPage: React.FC<Props> = (props) => {
         MFAcheckbox,
         mfa,
       })
-
+      console.log(`user authenticated with token: ${unscopedToken}`)
```
Comment (patil-pratik-87) [inline src/app/core/public/LoginPage.tsx:179]: console.log

File: src/app/core/public/LoginPage.tsx [hunk 2]
```diff
@@ -148,17 +153,15 @@ const LoginPage: React.FC<Props> = (props) => {
...
-    [],
-  )
-
+  const updateValue = (key) => (event) => {
+    updateParams({ [key]: event.target.value })
+  }
   const performLogin = useCallback(
     async (event) => {
       event.preventDefault()
+      console.log('settingLoadingState')
```
Comment (patil-pratik-87) [inline src/app/core/public/LoginPage.tsx:162]: console.log

File: src/app/core/public/LoginPage.tsx [hunk 3]
```diff
@@ -215,6 +218,7 @@ const LoginPage: React.FC<Props> = (props) => {
     [],
   )
 
+  console.log(`[render]: ${loading} ${loginUsername}`)
```
Comment (patil-pratik-87) [inline src/app/core/public/LoginPage.tsx:221]: console.log

File: src/app/core/public/LoginPage.tsx [hunk 4]
```diff
@@ -131,7 +131,12 @@ const LoginPage: React.FC<Props> = (props) => {
   const { ssoEnabled, onAuthSuccess, customerTier } = props
   const { history } = useReactRouter()
   const { getUserPrefs } = useScopedPreferences()
-  const { params, updateParams } = useParams(defaultParams)
+  const [params, setParams] = useState(defaultParams)
+
+  // const { params, updateParams: _updateParams } = useParams(defaultParams)
```
Comment (jruddell) [inline src/app/core/public/LoginPage.tsx:136]: be sure to remove the commented code as well

Comment (Nalinmujoo) [reply src/app/core/public/LoginPage.tsx:136]: Hi, I haven't pushed the final commit yet, a random warning came up, was testing what it was

File: src/app/core/public/LoginPage.tsx (hunk 3 above)
Comment (Nalinmujoo) [reply src/app/core/public/LoginPage.tsx:221]: Hi, I haven't pushed the final commit yet

File: src/app/core/public/LoginPage.tsx (hunk 2 above)
Comment (Nalinmujoo) [reply src/app/core/public/LoginPage.tsx:162]: Hi, I haven't pushed the final commit yet

File: src/app/core/public/LoginPage.tsx (hunk 1 above)
Comment (Nalinmujoo) [reply src/app/core/public/LoginPage.tsx:179]: Hi, I haven't pushed the final commit yet

**Review by jruddell — APPROVED** [review]

**Pull request merged**